sdk/php/tests/SpecificationsTest.php

```diff
@@ -78,8 +78,6 @@ public function testGetFieldsSpecification(): void
             'tech_data.manufacturer.name',
             'tech_data.brand.name.rus',
             'tech_data.transmission.type',
-            'insurance.osago.items[].date.start',
```

Comment:
Removed because now this fieds are fillable at least `ramiosago.base.ext` source